src/sentry/snuba/metrics/query_builder.py

```diff
@@ -644,10 +652,21 @@ def translate_results(self):
                 else:
                     operation = None
                     metric_name = key
+
                 if (operation, metric_name) not in self._query_definition_fields_set:
                     if totals is not None:
                         del totals[key]
                     if series is not None:
                         del series[key]
+                else:
+                    reversed_mri_key = (
+                        f"{get_reverse_mri(metric_name)}"
```

Comment:
nit: `metric_name` is actually an MRI at this point, right? Should we rename the variable to `metric_mri`?

Reply:
Yep agreed as well

Reply:
If it's cool with you, I'd rather do this particular renaming in a followup PR